modules/core/src/main/java/org/apache/ignite/internal/client/thin/ClientComputeImpl.java

```diff
@@ -115,6 +119,11 @@
         return executeAsync0(taskName, arg, dfltGrp, (byte)0, 0L);
     }
 
+    /** {@inheritDoc} */
+    @Override public <T, R> IgniteClientFuture<R> executeAsync2(String taskName, @Nullable T arg) throws ClientException {
```

Comment:
Do we need `executeAsync` at all?

Reply:
See above, I think we do, because of binary compatibility.